docs/NewProject.md

```diff
@@ -2,43 +2,60 @@
      Licensed under the MIT License. -->
      
 ## Adding a new project
-Do the following when adding a new project:
 
-### For all projects
-1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
-2. Right-click on the project and select Properties.
+### All project types
+
+1. Add your new project to the Axe.Windows Solution (`src\AxeWindows.sln`).
+2. Before creating a pull request, verify that Visual Studio can successfully load and build the entire solution in both Debug and Release.
+3. If your project requires NuGet dependencies which need to be installed along side it, update the <dependencies> section of `./src/ci/axe.windows.nuspec`.
+
+### For all .Net Framework projects
+
+1. Right-click on the project and select Properties.
 2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
-   - Currently .NET Framework 4.7.1 is the target. 
+   - Currently .NET Framework 4.7.1
```

Comment:
>Currently .NET Framework 4.7.1 [](start = 5, length = 30)

I get the intent of this line, but the phrasing feels like "pigeon English"

Reply:
It's not actually pigeon English; it's just punctuated incorrectly. I kept the original formatting initially, but I'll change it.

Reply:
Mostly it's the punctuation that makes this awkward. I'll fix it.